pages/configuration/https.rst

```diff
@@ -12,12 +12,25 @@ You also need to make sure that you have proper certificates in place, which are
 
 .. note:: If you're operating a single-node setup and would like to use HTTPS for the Graylog web interface and the Graylog REST API, it's possible to use :ref:`NGINX or Apache as a reverse proxy <configuring_webif_nginx>`.
 
+Thinks to consider
+==================
+
+You have multiple options to ensure that your connection is secure and save. The first would be to create a self-signed certificate, add that to the previous  copied java keystore and use this keystore with your Graylog java options.
+As you would need todo this for every certificate and every truststore this becomes unmaintenable in any form of cluster setup. Each node need to know all certificates of all nodes.
```

Comment:
Try this instead:
Since you will need to do this for every certificate and every trust store, this quickly becomes unmanageable in a clustered architecture. Each node needs to trust all certificates from all other nodes.